tests/test_utilities.py

```diff
@@ -252,7 +282,11 @@ def test_bounds():
     assert_raises(TypeError, galsim.BoundsD, 11, 23, 9, 12, 59)
     assert_raises(TypeError, galsim.BoundsD, xmin=11, xmax=23, ymin=17, ymax=50, z=23)
     assert_raises(TypeError, galsim.BoundsD, xmin=11, xmax=50)
-    assert_raises(ValueError, galsim.BoundsD, 11, 23, 17, "blue")
+    if is_jax_galsim():
+        # jax doesn't raise for this
+        pass
```

Comment:
I think I can guess what it does with floats.  But I'm quite curious what it does with "blue".